spinnman/transceiver.py

```diff
@@ -1313,6 +1335,10 @@ def execute(
             wait=False, is_filename=False):
         """ Start an executable running on a single chip
 
+        This method is currently deprecated and untested as there is no
+        known use. Same functionaility provided by ybug and bmpc.
+        Retained in case needed for hardware debugging.
+
```

Comment:
This is more of a utility method too; the same thing can be done in a slightly more complex way with execute_flood and a single chip.

Reply:
Marked likely to be removed.